### PR TITLE
Add create new user

### PR DESCRIPTION
**Pull request by ValentasK**

### CODE REVIEW

File: API.V1/Controllers/OrdersController.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public async Task<IActionResult> Get(int orderId)
         }
 
         [HttpPost]
-        public async Task<IActionResult> CreateOrder([FromBody] CreateCustomerOrderCommand command)
+        public async Task<IActionResult> CreateCustomer([FromBody] CreateCustomerOrderCommand command)
```
Comment (KestasMil) [inline API.V1/Controllers/OrdersController.cs:40]: Kode cia vadinas CreateCustomer bet parametro tipas apie customer order kalba ir kode variable order? Tai cia create customer ar create order endpointas? Me confused

File: API.V1/Handlers/CreateCustomerHandler.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+using API.V1.Responses;
+using Dapper;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Options;
+
+namespace API.V1.Handlers
+{
+    public class CreateCustomerHandler : IRequestHandler<CreateCustomerOrderCommand, CustomerResponse>
```
Comment (KestasMil) [inline API.V1/Handlers/CreateCustomerHandler.cs:12]: Neturetu cia but CreateCustomerOrderHandler arba CreateOrderHandler?

File: API.V1/Queries/sqlQueries.cs [hunk 3]
```diff
@@ -10,5 +13,15 @@ public static class sqlQueries
...
             "where p.Id = @Id;";
+
+        public static string CreateCustomer =
+            "BEGIN TRANSACTION " +
+            "DECLARE @PhoneId int; " +
+            "INSERT INTO [dbo].[Phone] ([PhoneNumber],[IsDeleted]) " +
+            "VALUES ( @PhoneNumber, 0); " +
+            "SELECT @PhoneId = scope_identity(); " +
+            "INSERT INTO [dbo].[Person] ([FirstName],[LastName],[PhoneId]) " +
+            "VALUES (@FirstName,@LastName, @PhoneId);" +
```
Comment (KestasMil) [inline API.V1/Queries/sqlQueries.cs:24]: trint lauk tas 19 ir 22 eilutes ir tada 24 eilutej tiesiog rasai:
VALUES (@FirstName,@LastName, scope_identity())

**Review by KestasMil — CHANGES_REQUESTED** [review]

File: API.V1/Controllers/OrdersController.cs (hunk 1 above)
Comment (ValentasK) [reply API.V1/Controllers/OrdersController.cs:40]: Wo blin kaip cia pramazinau

Comment (ValentasK) [reply API.V1/Controllers/OrdersController.cs:40]: Fixed

File: API.V1/Handlers/CreateCustomerHandler.cs (hunk 2 above)
Comment (ValentasK) [reply API.V1/Handlers/CreateCustomerHandler.cs:12]: Fixed

File: API.V1/Queries/sqlQueries.cs (hunk 3 above)
Comment (ValentasK) [reply API.V1/Queries/sqlQueries.cs:24]: Jo makes sense, tik be reikalo sukuriu kimtamaji

Comment (ValentasK) [reply API.V1/Queries/sqlQueries.cs]: Fixed

File: API.V1/Controllers/OrdersController.cs [hunk 4]
```diff
@@ -37,7 +37,7 @@ public async Task<IActionResult> Get(int orderId)
         }
 
         [HttpPost]
-        public async Task<IActionResult> CreateOrder([FromBody] CreateCustomerOrderCommand command)
+        public async Task<IActionResult> CreateCustomer([FromBody] CreateCustomerCommand command)
         {
             var order = await _mediator.Send(command);
```
Comment (KestasMil) [inline API.V1/Controllers/OrdersController.cs:42]: var order ??? neturetu but var customer?

**Review by KestasMil — CHANGES_REQUESTED** [review]

File: API.V1/Controllers/OrdersController.cs (hunk 4 above)
Comment (ValentasK) [reply API.V1/Controllers/OrdersController.cs:42]: Is kur tu jous trauki

Comment (ValentasK) [reply API.V1/Controllers/OrdersController.cs]: Fixed

**Review by KestasMil — APPROVED** [review]

**Pull request merged**